CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Fixed an issue where photos and videos could not be uploaded.
```

Comment:
Not sure what we'll do with this on the release. Or is it automatic? Do we add a new section with the version and build number? Do we have a fastlane lane for this?

Reply:
I don't think we have a procedure for this. I think we should go ahead and make a release ticket and branch for 0.1.16. We can have this PR target `release/0.1.16`. It will create some extraneous testflight versions of the production app but that's ok.

Reply:
@mplorentz are you suggesting we merge this to `main`, then create a new regular release? There's not much in `main` since 0.1.15, so I think that makes sense.

Reply:
I've updated the target branch to `main` so we're ready to do that.

Reply:
I think from now one we should be using the second number, so we have the patch number (the last one) for cases like this one. Also, we should be 0.1 anymore, Nos deserves more.

Reply:
@joshuatbrown no I was suggesting this PR targets `release/0.1.16`, because the point of doing a hotfix instead of a normal release is that you are excluding all other changes besides the emergency fix in case they introduce regressions and delay the release. But if there isn't much else on `main` then I'm fine with merging this and doing a normal release.